charts/tsa/templates/tsa-configmap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "tsa.config" . }}
+{{ include "tsa.namespace" . | indent 2 }}
+  labels:
+    {{- include "tsa.labels" . | nindent 4 }}
+data:
+  {{- if (eq .Values.server.args.signer "tink")}}
+  chain.pem: {{.Values.server.args.cert_chain | quote }}
+  tink.keyset.enc: {{.Values.server.args.tink_enc_keyset | quote }}
```

Comment:
The plan is to load this in from an external secret. Is this the right way to do this?

Reply:
agree with @hectorj2f

Reply:
+1 we already do that here, both env & mounting secret as a file, so I reckon using that pattern for everything would be nice.

https://github.com/sigstore/helm-charts/pull/413/files#diff-069f90964c38d9b0f1be8d1033a0e059beb91ad76e822fcfee88e611a199e235R55